core/src/lib.rs

```diff
@@ -28,6 +29,10 @@ mod test_help;
 
 pub(crate) use temporal_sdk_core_api::errors;
 
+pub use ephemeral_server::{
+    EphemeralExe, EphemeralExeVersion, EphemeralServer, TemporaliteConfig,
+    TemporaliteConfigBuilder, TestServerConfig, TestServerConfigBuilder,
+};
```

Comment:
Just noticed this - let's just pub the module rather than each thing individually. I think this one is nice to have behind a namespace.

Also ideally `EphemeralExe` and `EphemeralExeVersion` don't need to be pub, since they're largely an internal detail.

Reply:
> let's just pub the module rather than each thing individually

Ok

> Also ideally EphemeralExe and EphemeralExeVersion don't need to be pub, since they're largely an internal detail.

They are used for configuration. They are needed for a user to choose existing path, download default, or download specific version (I already use them from Python).

Reply:
Made the module public

Reply:
Ah, sure, makes sense